tests/flow/test_defrag.py

```
@@ -161,3 +162,136 @@ def test_frag_ratio(self):
+                            "MERGE (n:Item {id: row.id}) "
+                            "SET n.name = row.name, n.value = row.value",
+                            {"rows": [
+                                {"id": thread_id * 1000 + i,
+                                 "name": f"t{thread_id}_{i}",
+                                 "value": float(i)},
+                                {"id": thread_id * 1000 + i + 1,
+                                 "name": f"t{thread_id}_{i+1}",
+                                 "value": float(i + 1)},
+                            ]}
+                        )
+                    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)